src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReader {
+    public List<String> read(String filePath);
```

Comment:
The `public` modifier is redundant in interface method declarations. In an interface, all methods are public by default.